Userland/Libraries/LibGL/CMakeLists.txt

```diff
@@ -27,3 +27,12 @@ set(GENERATED_SOURCES
 
 serenity_lib(LibGL gl)
 target_link_libraries(LibGL PRIVATE LibGfx LibGLSL LibGPU)
+
+# Install symlinks at some common locations so ports can find LibGL without issue
+if (NOT BUILD_LAGOM)
```

Comment:
Should check `if (SERENITYOS)` instead. that comes from our Platform module

Reply:
Right, thanks!